src/components/JoinUs/index.test.jsx

```diff
@@ -7,7 +7,6 @@ jest.mock("react-i18next", () => ({
 }));
 
 const isTesting = true;
-
```

Comment:
Why did you delete this line ?

Reply:
i didnt delete
![image](https://user-images.githubusercontent.com/61707093/99191945-1914f380-2781-11eb-9de8-ada8de805212.png)

Reply:
there was empty line